parser/prism/Translator.cc

```diff
@@ -1314,13 +1314,13 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
         case PM_IN_NODE:                // An `in` pattern such as in a `case` statement, or as a standalone expression.
         case PM_PINNED_EXPRESSION_NODE: // A "pinned" expression, like `^(1 + 2)` in `in ^(1 + 2)`
         case PM_PINNED_VARIABLE_NODE:   // A "pinned" variable, like `^x` in `in ^x`
+        case PM_IMPLICIT_NODE:
```

Comment:
This is also used in Hash literals like `{ k: }`, and I don't see where that's handled. Is tha a feature we're missing?

https://ruby.github.io/play-ruby/?action=syntax%2Bprism&code=h+%3D+%7B+k%3A+%7D

Reply:
Sorry I misunderstood it and that's a good catch 👍 I've updated the PR to support it.